local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -536,6 +541,20 @@ void LocalPlannerNode::publishWaypoints() {
   original_wp_pub_.publish(sphere1);
   adapted_wp_pub_.publish(sphere2);
   smoothed_wp_pub_.publish(sphere3);
+  waypoint_pub_.publish(result.path);
+  path_pub_.publish(path_actual_);
+  publishSetpoint(result.velocity_waypoint, result.waypoint_type);
+
+  // to mavros
+  mavros_msgs::Trajectory obst_free_path = {};
+  if(local_planner_.use_vel_setpoints_){
+      mavros_vel_setpoint_pub_.publish(result.velocity_waypoint);
```

Comment:
I think the transformation from `world` to `local_origin` is missing.

Reply:
As discussed on slack: I changed all the frames to world frame to have a consistent use of frames